pkg/roachpb/api.proto

```diff
@@ -1333,6 +1333,15 @@ message ExternalStorage {
   FileTable FileTableConfig = 8 [(gogoproto.nullable) = false];
 }
 
+// ExternalStorageRetryTracingEvent is the trace recording used to track retries
+// related to external storage operations.
+message ExternalStorageRetryTracingEvent {
```

Comment:
This proto will be used by https://github.com/cockroachdb/cockroach/pull/65725/commits/b19b02f996eea091dfa9c83bd852a32caf48d79b#diff-ca58a10cde4cd7f7a6b9650155472d79d3a2734f4a82108ed6a8f786a9aca109R1336 as well, but this was a smaller change so can be checked in sooner.